### ISSUE TITLE
mongodb not installed by installation script

### ISSUE THREAD

**Issue by dirkk0** [sandstorm-io/sandstorm#135]
Hi,

  I read somewhere in the issues that mongodb will be installed by the installation script; this wasn't true for me. Surprisingly enough sandstorm nevertheless ran. But mongo wan't there, I needed to install it manually.

Best,
  Dirk

**Comment by qcu** [comment]
Hi @dwrensha, Does this comment still hold true, in 2016?

I don't see any updates to [Kenton's fork](https://github.com/kentonv/mongo/tree/niscu) since 2014.

> Note that if you want to use Mongo from within a Sandstorm app that you're developing, you probably should use Kenton's fork, which you will need to install separately.

I'm trying to package a non-meteor Node application that depends on mongodb, and just looking for current best practices for mongodb with a non-meteor app.

This looks like a promising start...

> David Renshaw wrote: you may find it useful to look at my start script for ShareLaTeX, which also needs a Mongo database: https://github.com/dwrensha/sharelatex/blob/sandstorm-app/start.bash

**Comment by dwrensha** [comment]
We currently do not recommend that you use Niscu. Instead, we recommend that you use Mongo 3.0 with the wiredTiger storage engine, which, unlike mmapv1, does not need to allocate tens of megabytes right of the bat. The [meteor-spk startup script](https://github.com/sandstorm-io/meteor-spk/blob/master/start.js) and the [duoludo Mongo configuration](https://github.com/dwrensha/duoludo/blob/master/.sandstorm/mongod.wiredTiger.conf) should be good references.

**Comment by qcu** [comment]
Thank you, @dwrensha. 

FWIW, I collected my rough notes regarding the approach to incorporating mongodb into a sandstorm package within this gist: [HOWTO Sandstorm diy app package including mongodb 3.0](https://gist.github.com/qcu/73fe91e667ba590c8bf107ba96242655).

However, I discovered that mogodb was optional in the [CryptPad fork](https://github.com/qcu/cryptpad/tree/sandstorm) that I am experimenting with packaging for sandstorm. So, I haven't fully tested the mongodb approach, yet.
